extensions-core/druid-bloom-filter/src/main/java/org/apache/druid/query/aggregation/bloom/BloomFilterMergeBufferAggregator.java

```diff
@@ -0,0 +1,128 @@
+/*
```

Comment:
There's quite a lot of duplicate or similar code between this and BloomFilterBufferAggregator. Any opportunity to consolidate?

Reply:
Consolidated common code into `BaseBloomFilterAggregator` and `BaseBloomFilterBufferAggregator`